### ISSUE TITLE
residential_quota_downselect for single family detached buildstock.csv

### ISSUE THREAD

**Issue by priyada7** [NREL/resstock#1221]
I had a query in using Resstock. I am new to this tool, and facing issue trying to understand.

Basically, I want to build housing chacteristics of 100 homes in climate zone 1A of single family detached. To use this I use the residential_quota_downselect 
and below is the code snippet in yml file.

But the csv that it generates as buildstock.csv has all regions and all types of houses and the results-csv status comes as fail.

Can you please help me here? 

---
sampler:
  type: residential_quota_downselect
  args:
    n_datapoints: 100
    logic:
      - Geometry Building Type RECS|Single-Family Detached
      - site_iecc_zone|1A
    resample: false

**Comment by joseph-robertson** [comment]
The [filtering logic](https://buildstockbatch.readthedocs.io/en/stable/filtering_logic.html#filtering-logic) should be of the form "parameter|option". So in your example, instead of "site_iecc_zone|1A", you'd want "ASHRAE IECC Climate Zone 2004|1A".

**Comment by priyada7** [comment]
okay!! but still the problem persists.
[buildstock.csv](https://github.com/NREL/resstock/files/14643726/buildstock.csv)

**Comment by priyada7** [comment]
Is it because the resample is set to 'false'? I am not geeting the desired houses? I did change it to 'true' with below syntax :

sampler:
  type: residential_quota_downselect
  args:
    n_datapoints: 350000
    logic:
      - Geometry Building Type RECS|Single-Family Detached
      - ASHRAE IECC Climate Zone 2004|1A
    resample: true
and the error I received said "Not supporting residential_quota_downselect's 'resample' at this time."

**Comment by joseph-robertson** [comment]
Sounds like you are using `run_analysis.rb`. Currently, `resample` is not supported when using `run_analysis.rb`. See the note in https://resstock.readthedocs.io/en/latest/basic_tutorial/run_project.html#using-run-analysis-rb about that.

It is expected that your buildstock.csv contains _all_ samples. The downselect is applied at simulation time; the results.csv should show "Invalid" completed status (with no simulation results) for samples that aren't both Single-Family Detached and IA.

**Comment by joseph-robertson** [comment]
Yes. You could generate the full buildstock.csv, manually downselect its rows (e.g., using Excel filters), re-enumerate the Building column, and then using the [precomputed sampler](https://buildstockbatch.readthedocs.io/en/stable/samplers/precomputed.html) type that points to the downselected sample file.

**Comment by nmerket** [comment]
If you use [buildstock_local](https://buildstockbatch.readthedocs.io/en/stable/run_sims.html#local) from buildstockbatch, the system we use to run larger analyses, you can do the downsample you're looking for.
